libarchive/Makefile

```diff
@@ -1,19 +1,20 @@
 CFLAGS     = -g
-INCLUDE    = -I/usr/include -I.
-LIBS       = -larchive
+INCLUDE    = -I/usr/local/include -I/usr/include -I.
+LIBS       = -L /usr/local/lib -larchive
 
-PYVER ?= 2.7
+#DEFINES = -DPyUnicode_AS_STRING=PyUnicode_AS_DATA
```

Comment:
Can this be removed and the place below that it was used?

Reply:
Done